src/v/kafka/group_probe.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "config/configuration.h"
+#include "prometheus/prometheus_sanitize.h"
+
+#include <seastar/core/metrics.hh>
+
+namespace kafka {
+class group_offset_probe {
+public:
+    explicit group_offset_probe(model::offset& offset) noexcept
+      : _offset(offset) {}
+
+    void setup_metrics(
+      const kafka::group_id& group_id, const model::topic_partition& tp) {
+        namespace sm = ss::metrics;
+        auto group_label = sm::label("group");
+        auto topic_label = sm::label("topic");
+        auto partition_label = sm::label("partition");
+        std::vector<sm::label_instance> labels{
```

Comment:
nice! do we need to also include `namespace` from the ntp?

Reply:
I think, we don't.
Group doesn't store namespace

Reply:
ah, good point. I guess we don't need the namespace!